src/averageAges.js

```diff
@@ -14,12 +14,18 @@
  *
  * @return {number}
  */
+function calculateAverage(people) {
+  const sum = people.reduce((prev, person) =>
+    (person.died - person.born) + prev, 0);
+
+  return sum / people.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm'
+  && (century ? Math.ceil(person.died / 100) === century : true));
```

Comment:
The ternary operator `century ? Math.ceil(person.died / 100) === century : true` can be simplified by removing the `: true` part. This will make the code more readable.